e2e/cli-plugins/help_test.go

```diff
@@ -41,9 +41,9 @@ func TestGlobalHelp(t *testing.T) {
 		regexp.MustCompile(`^A self-sufficient runtime for containers$`),
 		regexp.MustCompile(`^Management Commands:$`),
 		regexp.MustCompile(`^  container\s+Manage containers$`),
+		helloworldre,
```

Comment:
I suggest to bookend this with something which comes after in the mgmt commands too, just as a small sanity check on the ordering, e.g. `^  image\s+Manage images$` would be suitable.

You could drop either `create` or `ps` from below too, since the need for both was the same purpose before this change.

Reply:
Added 👍